.github/workflows/ci.yaml

```diff
@@ -5,56 +5,60 @@ on:
     - cron:  '0 0 * * *'
 jobs:
   test:
-    name: PHPUnit (PHP ${{ matrix.php-versions }}) (${{ matrix.composer.name }} dependencies)
+    name: PHPUnit (PHP ${{ matrix.php-versions }}) (${{ matrix.composer }} dependencies)
     runs-on: ${{ matrix.operating-system }}
     strategy:
       fail-fast: false
       matrix:
         operating-system: [ubuntu-latest]
         php-versions: ['8.1', '8.2']
```

Comment:
can you add 8.3?

Reply:
I want to keep this PR as small as possible, I would be happy to follow this up with another PR that adds PHP 8.3 and 8.4

Reply:
ok